### PR TITLE
Doc alaro sfx

### PR DESCRIPTION
**Pull request by ddegrauwe**
Documenting the work from the DAVAI Working Week to create an ALARO+SURFEX test.

### CODE REVIEW

File: docs/src/exercise4developers.md [hunk 1]
```diff
@@ -1 +1,119 @@
...
+### Adding an ALARO+SURFEX test to DAVAÏ
+
+This section describes what was done to add an ALARO+SURFEX test to DAVAÏ. It may serve as a recipe to add other tests.
+
+First, create a new DAVAÏ experiment with `davai-new_xp`. Also, run following commands to set the environment:
+
+```
+source ~acrd/.vortexrc/profile
+cp ~rm9/.vortexrc/uget-client-defaults.ini .vortexrc/
+```
```
Comment (roelstappers) [inline docs/src/exercise4developers.md]:
````suggestion
```bash
source ~acrd/.vortexrc/profile
cp ~rm9/.vortexrc/uget-client-defaults.ini .vortexrc/
```
````

File: docs/src/exercise4developers.md [hunk 2]
```diff
@@ -1 +1,119 @@
...
+
+Next, initialize the hack directory for your user:
+
+```
+uget.py bootstrap_hack ${USER}
+```
```
Comment (roelstappers) [inline docs/src/exercise4developers.md]:
````suggestion
```bash
uget.py bootstrap_hack ${USER}
```
````

File: docs/src/exercise4developers.md [hunk 3]
```diff
@@ -1 +1,119 @@
...
+* add a section for the model:
+```
+[alaro]
+model               = alaro
+LAM                 = True
+input_shelf         = &{input_shelf_lam}
+fcst_term           = 12
+expertise_term      = 12
+coupling_frequency  = 3
+```
```
Comment (roelstappers) [inline docs/src/exercise4developers.md]:
````suggestion
  ```ini
  [alaro]
  model               = alaro
  LAM                 = True
  input_shelf         = &{input_shelf_lam}
  fcst_term           = 12
  expertise_term      = 12
  coupling_frequency  = 3
  ```
````

File: docs/src/exercise4developers.md [hunk 4]
```diff
@@ -1 +1,119 @@
...
+
+* add a section for the forecast itself:
+```
+[forecast-alaro1_sfx-chmh2325]
+alaro_version       = 1_sfx
+rundate             = date(2021022000)
+```
```
Comment (roelstappers) [inline docs/src/exercise4developers.md]:
````suggestion
  ```ini
  [forecast-alaro1_sfx-chmh2325]
  alaro_version       = 1_sfx
  rundate             = date(2021022000)
  ```
````

File: docs/src/exercise4developers.md [hunk 5]
```diff
@@ -1 +1,119 @@
...
+
+* since we're using a new domain (chmh2325), add a section for this domain:
+```
+[chmh2325]
+geometry            = geometry(chmh2325)
+timestep            = 90
+```
```
Comment (roelstappers) [inline docs/src/exercise4developers.md]:
````suggestion
  ```ini
  [chmh2325]
  geometry            = geometry(chmh2325)
  timestep            = 90
  ```
````

File: docs/src/exercise4developers.md [hunk 6]
```diff
@@ -1 +1,119 @@
...
+
+##### Modifications to the file `tasks/forecasts/standalone_forecasts.py`:
+
+The easiest is to copy and modify an existing forecast. In this case, we added to the following to the `alaro` family:
+```
+                    Family(tag='chmh2325', ticket=t, nodes=[
+                          StandaloneAlaroForecast(tag='forecast-alaro1_sfx-chmh2325', ticket=t, **kw),
+                    , **kw),
+```
```
Comment (roelstappers) [inline docs/src/exercise4developers.md]:
````suggestion
```python
                    Family(tag='chmh2325', ticket=t, nodes=[
                          StandaloneAlaroForecast(tag='forecast-alaro1_sfx-chmh2325', ticket=t, **kw),
                    , **kw),
```
````

**Review by roelstappers — COMMENTED** [review]
Thanks @ddegrauwe. Very useful to have the alaro example. I added some small suggested changes. In a bullet point list with a code block, the code block needs to be indented to the same level as the text to get proper alignment e.g. 
````
* Some text 
  ```
  code 
  ```
````

In the github gui you should be able to accept the suggested changes.

**Comment by roelstappers** [comment]
Perhaps best to squash merge this PR @AlexandreMary

**Comment by AlexandreMary** [comment]
Squashed and merged.
@ddegrauwe do you have the alaro+surfex branch of DAVAI-tests available btw ?

**Pull request closed without merging**